__tests__/patch-state.spec.ts

```diff
@@ -46,48 +46,69 @@ describe('view state & patch state', () => {
       ],
     ]);
 
-    test(`View state${process.env.NEAR_RUNNER_NETWORK !== 'testnet' ? '' : '(skipping on testnet)'}`, async () => {
+    test(`View state${
+      process.env.NEAR_RUNNER_NETWORK !== "testnet"
+        ? ""
+        : "(skipping on testnet)"
+    }`, async () => {
```

Comment:
Test-skipping now explained above

```suggestion
    test("View state", async () => {
```